packages/react-native-gesture-handler/src/web/handlers/GestureHandler.ts

```diff
@@ -369,7 +370,7 @@ export default abstract class GestureHandler implements IGestureHandler {
     const { onGestureHandlerEvent, onGestureHandlerTouchEvent }: PropsRef =
       this.propsRef!.current;
 
-    const touchEvent: ResultTouchEvent | undefined =
+    const touchEvent: ResultEvent<TouchEvent> | undefined =
```

Comment:
Are you sure that `TouchEvent` will have correct fields and won't expose something that we do not provide?